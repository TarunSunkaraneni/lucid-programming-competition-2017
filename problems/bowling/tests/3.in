20
9 0 8 1 9 1 9 0 7 3 7 2 6 0 7 3 9 1 7 2
10 6 3 7 0 9 1 3 6 1 7 6 1 5 3 8 1 8 1
3 7 3 5 6 4 6 2 5 5 9 1 9 1 10 8 2 8 0
7 0 3 7 8 2 9 1 5 4 10 7 0 9 1 4 2 8 1
9 1 7 3 10 9 0 10 9 1 10 5 3 9 1 9 1 10
1 5 9 1 6 1 2 1 6 1 8 0 5 3 9 1 7 3 5 5 8
2 8 7 1 10 6 2 6 3 5 4 3 6 4 3 6 4 3 7 7
10 0 10 3 3 10 9 1 6 3 3 5 9 1 4 4 10 6 4
9 1 8 2 2 6 4 4 6 2 4 3 8 0 8 2 10 10 5 4
10 10 5 5 5 3 8 2 9 1 9 0 5 2 1 8 5 4
5 3 10 10 4 3 4 5 3 4 3 7 9 1 7 3 1 7
8 1 3 4 7 2 7 3 7 3 7 3 9 1 6 1 8 1 6 2
8 0 7 3 7 3 3 6 8 2 6 3 8 0 4 4 7 1 4 5
6 4 10 4 3 8 2 2 6 4 3 3 7 9 1 10 10 7 3
10 8 2 6 2 5 5 8 1 7 2 3 4 7 3 6 1 7 2
7 2 5 3 5 3 7 2 9 1 8 0 10 1 2 4 5 9 1 9
8 1 2 7 6 1 3 7 1 0 1 9 4 4 9 1 3 4 9 0
8 2 9 1 8 1 3 6 1 5 3 5 10 3 7 10 6 1
10 6 4 9 1 7 2 3 7 10 1 5 8 1 6 2 10 9 0
3 5 10 6 3 4 2 9 1 8 1 9 1 7 2 9 1 10 4 5
