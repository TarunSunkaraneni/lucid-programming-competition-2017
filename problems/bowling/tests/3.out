123
98
149
121
170
105
115
134
133
140
127
118
105
157
119
104
88
133
143
134
